### PR TITLE
adding lint rule for console.log

### PR DESCRIPTION
**Pull request by shogunpurple**
## Description
We committed a `console.log` debugging statement to prod and it showed up for users. This lint rule prevents this from happening in future as CI will fail the lint check if there's any `console.log` statements lying around in the frontend codebase.

### CODE REVIEW

**Review by melohagan — APPROVED** [review]

File: .eslintrc.json [hunk 1]
```diff
@@ -45,6 +45,12 @@
         "no-prototype-builtins": "off",
         "local-rules/no-budibase-imports": "error"
       }
+    },
+    {
+      "files": ["packages/builder/**/*"],
```
Comment (mike12345567) [inline .eslintrc.json:50]: We should probably apply this to the client library as well!

**Review by mike12345567 — APPROVED** [review]
LGTM - would just add linting to client as well!

File: .eslintrc.json (hunk 1 above)
Comment (shogunpurple) [reply .eslintrc.json:50]: I have actually already pushed an update with that, but github hasn't processed it yet. Super weird.

**Review by aptkingston — APPROVED** [review]
LGTM!

**Review by deanhannigan — APPROVED** [review]
LGTM

**Pull request merged**